### DISCUSSION TITLE
Feature: Moving pods meta boxes above editor (or ability to set metabox position)

### DISCUSSION THREAD

**Discussion by xkungfu** [pods-framework/pods#6730] [Ideas]
how to put a pods meta box above the editor but below the title section on the edit post page?

IDEAL:

Give the ability to determine _where_ on the post editor screen that the Pods Metabox is placed.

**Comment by jimtrue** [comment]
@xkungfu You ended up using the following code to move the existing meta boxes above the editor:

```
function foo_move_deck() {
       # Get the globals:
       global $post, $wp_meta_boxes;

       # Output the "advanced" meta boxes:
       do_meta_boxes( get_current_screen(), 'normal', $post );

       # Remove the initial "advanced" meta boxes:
       unset($wp_meta_boxes['post']['normal']);
   }

add_action('edit_form_after_title', 'foo_move_deck');
```

**Comment by jimtrue** [comment]
Changing focus of this issue to ability to control where the Meta Box is placed. Even when we add Field Groups #799, is this going to allow us to control where they go?
